build/build.sh

```diff
@@ -82,9 +83,12 @@ if [ "${BUILD_PDF:-}" != "false" ] && [ -z "$DOCKER_EXISTS" ]; then
   rm images
 fi
 
-# Create PDF output (unless BUILD_PDF environment variable equals "false")
+# If Docker is available, use athenapdf to create PDF
 if [ "${BUILD_PDF:-}" != "false" ] && [ -n "$DOCKER_EXISTS" ]; then
   echo >&2 "Exporting PDF manuscript using Docker + Athena"
+  if [ "${CI:-}" = "true" ]; then
```

Comment:
Should we add a comment here explaining why the delay is longer when running on CI?  It is a little counter intuitive.  One might expect that the delay should always be long enough to avoid mathjax problem regardless of whether we build locally or remotely.

That said, the decision does make sense to me.  We are less sensitive to build time when running on Travis CI.  We also want the "official" versions of the manuscripts hosted on GitHub pages to be less likely to contain errors.

Reply:
https://github.com/manubot/rootstock/pull/255/commits/8119a99b5317d169907b2aecb5c40ce3366c24a7